src/components/PoiItem.jsx

```diff
@@ -37,6 +38,9 @@ const PoiItem = React.memo(({ poi,
           <OpeningHour schedule={new OsmSchedule(poi.blocksByType.opening_hours)} />
         </div>}
       </div>
+      {inList && <div className="poiItem-address u-text--subtitle u-ellipsis">
+        <Address address={poi.address} inline omitCountry omitCity />
```

Comment:
A bit sad to use `omitCity` here :crying_cat_face:  
Is it due to the lack of space ?

Reply:
This is what the designs show, but I agree with you, it would be better with it. I think it's a matter of space and to avoid visual clutter. But IMO we should display it. Maybe I can merge smth with it first, and discuss it afterwards?

Reply:
I'll remove my last commit for now.

Reply:
Ok, thanks